source/sass/main.scss

```diff
@@ -428,6 +428,7 @@ button {
 
     &:hover {
       border: 2px solid darken($nsf-yellow, 10%);
```

Comment:
ah okay, that sounds odd to me. But agreed that it should be tackled in another PR.